nucleus/common/common-util/src/main/java/com/sun/appserv/util/cache/mbeans/JmxBaseCache.java

```diff
@@ -57,7 +57,7 @@ public class JmxBaseCache implements JmxBaseCacheMBean {
     private final BaseCache baseCache;
 
     public JmxBaseCache(BaseCache baseCache, String name) {
-        this.baseCache = baseCache;
+        this.baseCache = new BaseCache();
```

Comment:
Are you not providing the passed in BaseCache to newly constructed one purpose?

Reply:
Good catch